### DISCUSSION TITLE
【委托】求一个封面图

### DISCUSSION THREAD

**Discussion by GT025233** [share121/inter-knot#955] [其他]
哪个传奇绳匠可以给个这个图片的原图啊，谢谢！
![image](https://github.com/user-attachments/assets/1911bd41-9c2c-4aa5-bc02-136da8fccea5)

**Comment by capacitor1** [comment]
🤔[default-cover.oB27RXoN.svg](https://share121.github.io/inter-knot/_nuxt/default-cover.oB27RXoN.svg)
如果说的是网站上面的这张图，那么可以右键，另存为
